### PR TITLE
Initial spoot with side view!

### PR DESCRIPTION
**Pull request by NicholasLago**
Hey guys I redid the initial commit but side view style!

### CODE REVIEW

**Comment by iancawood** [comment]
Mind adding `.vs` to the `.gitignore`? I'll test the actual pr tonight or tomorrow.

**Comment by NicholasLago** [comment]
Added!

**Comment by iancawood** [comment]
In adding the stuff to the gitignore you actually have to also remove stuff that it is now ignoring. It's kinda dumb, but git. 

See this: https://stackoverflow.com/questions/1274057/how-to-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore

File: Assets/Scripts/Players/Movement.cs [hunk 1]
```diff
@@ -2,20 +2,51 @@
...
     }
 
-    // Update is called once per frame
     void Update () {
-        Vector2 moveDirection = new Vector2(
-            Input.GetAxis("Horizontal"),
-            Input.GetAxis("Vertical")
-            ); 
-        controller.Move(moveDirection * speed * Time.deltaTime);
+
```
Comment (iancawood) [inline Assets/Scripts/Players/Movement.cs:19]: Out of curiosity, why change from the simple `Input.GetAxis()` logic? Doesn't really matter in the long run. Was there it not working?

**Comment by iancawood** [comment]
It's good. Once this is in we can add more parts. So it might be worth just YOLO merging and then continue development from there.

**Pull request merged**